README.md

```diff
@@ -2,404 +2,204 @@
 
-# Installing
+# Supported versions
 
-``` console
-pip install boxsdk
-```
+To enhance developer experience, provide full Box API coverage, and rapid updates we have introduced the new generated codebase through the `box_sdk_gen` package.
+The `box_sdk_gen` package is available in two major supported versions: v4 and v10.
 
-The current version of the SDK is v3.x --- With this release support for
-Python 3.5 and earlier (including 2.x) has been dropped. if you're
-looking for the code or documentation for v1.5.x, please see the [1.5
-branch](https://github.com/box/box-python-sdk/tree/1.5).
-
-# Getting Started
+## Version v4
 
-To get started with the SDK, get a Developer Token from the
-Configuration page of your app in the [Box Developer
-Console](https://app.box.com/developers/console). You can use this token
-to make test calls for your own Box account.
-
-The SDK provides an interactive `DevelopmentClient` that makes it easy
-to test out the SDK in a REPL. This client will automatically prompt for
-a new Developer Token when it requires one, and will log HTTP requests
-and responses to aid in debugging and understanding how the SDK makes
-API calls.
-
-``` pycon
->>> from boxsdk import DevelopmentClient
->>> client = DevelopmentClient()
-Enter developer token: <ENTER DEVELOPER TOKEN HERE>
->>> user = client.user().get()
-GET https://api.box.com/2.0/users/me {'headers': {'Authorization': '---wXyZ',
-            'User-Agent': 'box-python-sdk-2.0.0',
-            'X-Box-UA': 'agent=box-python-sdk/2.0.0; env=python/3.6.5'},
-'params': None}
-"GET https://api.box.com/2.0/users/me" 200 454
-{'Date': 'Thu, 01 Nov 2018 23:32:11 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Strict-Transport-Security': 'max-age=31536000', 'Cache-Control': 'no-cache, no-store', 'Content-Encoding': 'gzip', 'Vary': 'Accept-Encoding', 'BOX-REQUEST-ID': '0b50luc09ahp56m2jmkla8mgmh2', 'Age': '0'}
-{'address': '',
-'avatar_url': 'https://cloud.app.box.com/api/avatar/large/123456789',
-'created_at': '2012-06-07T11:14:50-07:00',
-'id': '123456789',
-'job_title': '',
-'language': 'en',
-'login': 'user@example.com',
-'max_upload_size': 16106127360,
-'modified_at': '2018-10-30T17:01:27-07:00',
-'name': 'Example User',
-'phone': '',
-'space_amount': 1000000000000000.0,
-'space_used': 14330018065,
-'status': 'active',
-'timezone': 'America/Los_Angeles',
-'type': 'user'}
-
->>> print(f'The current user ID is {user.id}')
-The current user ID is 123456789
-```
+In v4 of the Box Python SDK, we are introducing a version that consolidates both the manual package (`boxsdk`)
+and the new generated package (`box_sdk_gen`). This allows developers to use both packages simultaneously within a single project.
+Version v4 of the Box Python SDK will be supported until 2027. During this period, the `boxsdk` package will receive only bug fixes and security patches.
```

Comment:
How about mentioning here that the package boxsdk is deprecated in code to enhance users to migrate to boxsdkgen package, and that they would receive a warning when using boxsdk?

Reply:
I would also add line break somewhere here 
<img width="853" height="237" alt="Screenshot 2025-10-17 at 09 42 40" src="https://github.com/user-attachments/assets/b93cf733-4868-463d-8b57-cf9dbc3fc9d8" />

Reply:
I added ` During this period, 
the `boxsdk` package will marked as deprecated and will receive only bug fixes and security patches.` but I wouldn't mention getting warning here